README.md

```diff
@@ -7,5 +7,18 @@ coming soon …
 ## Installation
 `meteor add space:accounts`
```

Comment:
What do you think about changing installation instructions on **develop** branches to mgp installation instead of this?